public/v4/apps/dufs.yml

```diff
@@ -0,0 +1,48 @@
+captainVersion: 4
+
+services:
+  '$$cap_appname':
+    caproverExtra:
+      dockerfileLines:
```

Comment:
https://github.com/sigoden/dufs?tab=readme-ov-file#with-docker

Looks like they have official Docker image for the app. Can we use that instead ?